docs/source/protocols/generic/label.rst

```diff
@@ -63,7 +66,11 @@ Label
 +------------------+-----------------+
 | color            | *any color*     |
 +------------------+-----------------+
+| blink            | "on" or "off    |
```

Comment:
Didn't i use backticks in the other pages? Also, you miss a `"` here.

Reply:
Shouldn't I use the "versionadded" approach here too?

I noticed that for protocols the values of the state option are shown as ``on / off``. To stay in line with that I think I can better change the ``"on" or "off"``  to ``on / off``. Do you agree?

Reply:
Yes, agreed. Also on the versionadded.

Reply:
Done